packages/components/src/components/FileUploader/utils/getStatus.ts

```diff
@@ -5,13 +5,16 @@ export const getStatus = ({
   progress,
   fileUrl,
   errorMessage,
+  required,
 }: {
   progress: number;
   fileUrl?: string;
   errorMessage?: string;
+  required?: boolean;
 }): FileUploaderStatus => {
   if (progress > 0 && progress < 100) return "loading";
   if (!isEmpty(fileUrl)) return "success";
   if (!isEmpty(errorMessage)) return "error";
+  if (!isEmpty(errorMessage) && required && !fileUrl) return "error";
```

Comment:
This seems unnecessary. It would never actually get to this line because it would always return at line 17 if there's an error message.  I don't think we really need this either, if there's an error message then it should just be an error state, we don't need to check the other properties here.

Reply:
jesus 🤦